app/charts/map/layer.tsx

```diff
@@ -11,7 +11,7 @@
 import assert from "assert";
 
 import isEqual from "lodash/isEqual";
-import { Map } from "maplibre-gl";
+import { Map } from "mapbox-gl";
```

Comment:
Why ? Don't we use maplibre ?

Reply:
In fact the type of the map retrieved from the events (onLoad, onMove, etc) is the type from MapBox and they are not the same :) In fact in this particular place there are no TS errors, but to properly type things in other places (as mapRefs) it had to be a type from MapBox.